peerprep-fe/src/app/(main)/page.tsx

```diff
@@ -1,13 +1,6 @@
 'use client';
 import MainComponent from './components/Main';
-import { useAuth } from '@/hooks/useAuth';
-import LoadingSpinner from '@/components/loading/LoadingSpinner';
 
 export default function Home() {
-  const { isLoading } = useAuth();
```

Comment:
yee makes sense to just have loading done in the question table using the skeleton